release_images.yml

```diff
@@ -5,14 +5,22 @@ release_images:
     version: "1.15.3"
     training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37", "py36"]
+      python_versions: ["py37", "py36", "py27"]
       os_version: "ubuntu18.04"
       cuda_version: "cu100"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
                             # to images being published.
       force_release: False  # [Default: False] Set to True to force images to be published even if the same image
                             # has already been published. Re-released image will have minor version incremented by 1.
+    inference:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py36", "py27"]
```

Comment:
Is there no "py37" option here?

Reply:
We only have currently py27 and py36 Images for Inferences for TF1.15.